### DISCUSSION TITLE
Is this where I complain that at least official Microsoft packages default to German if language is set to an East European country?

### DISCUSSION THREAD

**Discussion by mircboe** [microsoft/winget-cli#5076] [General]
Who had the great idea to lump us together? You think we all know German here in Eastern Europe because we're small and insignificant and the Germans are our economic saviors and geopolitical leaders? Yes, it’s the same kind of arrogance or cluelessness that led to poorly planned partitions in places like the Middle East, India, and Africa. The partitioning of these regions often ignored ethnic, religious, and linguistic differences, leading to long-term instability and suffering.

**Comment by denelon** [comment]
I don't have quite enough detail to point you in the right direction. The locale selection logic depends on what locales are supported by various packages, and the locale Windows is configured to use. Different "Packages"/"Apps"/"Programs" have different sets of locales supported in general. It depends on the publisher. For Microsoft products, it depends on if the product is targeting "all users" of the Windows OS, or if they are targeting a subset of users like "Developers".
